#----------echo_point2pose---------------
#time_interval_context_evaluation
time_interval: 10.0

#Object Set of Interest
Interest_of_Objects: ["cup", "bottle"]
#Interest_of_Objects: ["cup", "bottle", "book", "bowl"]



